test/integration/crcsuite/prepare.go

```diff
@@ -56,6 +58,60 @@ func DownloadBundle(bundleLocation string, bundleDestination string) (string, er
 	return filename, nil
 }
 
+func CopyFilesToTestDir() {
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		fmt.Printf("Error retrieving current dir: %s", err)
+		os.Exit(1)
+	}
+	pDir := path.Dir(pwd)
+	pDir = path.Dir(pDir)
+	pDir = path.Dir(pDir)
```

Comment:
@jsliacan What you want to simulate here ` cd ../..` like this?

Reply:
Exactly. I don't know the absolute path of the directory, and I don't want to use `/` as this should work on all platforms. Didn't find a way to go up the tree by _n_ steps at once... any suggestions welcome :)

Reply:
@jsliacan How about https://play.golang.org/p/Tkz7vPJ5rNq one?

Reply:
Maybe we could pass the path to testdata/ to the tests somehow rather than hardcoding it's `$(getcwd)/../../..`?

Reply:
> @jsliacan How about https://play.golang.org/p/Tkz7vPJ5rNq one?

`l := strings.Split(s, string(filepath.Separator))` can use `filepath.SplitList` instead.

Reply:
@cfergeau I think `SplitList` is for path list separator not for a single path. https://golang.org/pkg/path/filepath/#SplitList

Reply:
I think `filepath.SplitList` splits a list of paths not a path (e.g. based on `:` for Unix): https://golang.org/pkg/path/filepath/#SplitList.  As for passing the testdata dir to `make integration` -- tempting, but I worry that we're already passing way too many things. Plus this info is deduceable, just not in a nice way it seems. Thanks though, I'll change it to something and we'll see.

Reply:
Oops, sorry, read too quickly the go doc :)

Reply:
I changed it, but it's not much nicer than it was! Hmmm...